mxcube3/core/loginutils.py

```diff
@@ -332,17 +332,11 @@ def login_info(login_info):
     user = get_user_by_sid(session.sid)
 
     if user:
-        if res["loginType"].lower() != "user":
-            res["selectedProposal"] = user["loginID"]
-        elif proposal_info:
```

Comment:
we need this for the username based login

Reply:
But you still have it, like this is the same regardless, I was a bit puzzled by

```
elif proposal_info:
  ...
```
Because proposal_info is always undefined, its never assigned to

Reply:
What did you intend proposal_info to be ?

Reply:
I guess I added myself on https://github.com/mxcube/mxcube3/commit/c34f0a7e555df31e87af7fbb43a611b5356b85e0#diff-ac7ec31afc04b853184223e8a487bce4
But I see that it does not make sense since `proposal_info` is not defined,??
I think the line we need would be the first if clause, otherwise res["selectedProposal"] would not have the correct proposal but username.
Cannot test this now, but could you change your loginType in the mockups to 'user' and test it yourself?

Reply:
The proposal will be selected with a call from the client right after

Reply:
yes it must have been you who added that piece of code, well Im not overly enthusiastic about testing it but I will since you can't

Reply:
Can I just ask, what you are using today ?

Reply:
thanks.
using today? In what sense? We are not using this code in production, we are still behind. If thats what you asked

Reply:
Whats the logic that you are using today to select proposal and send this information ?

Reply:
Rigth after login, the client checks the response and looks for the `loginType === 'User'`, if so ProposalsForm form is displayed, there the user selects a proposal from the available ones for that user.  
That form calls then:
 actions.login.sendSelectProposal -> postProposal  -> API ../lims/proposal -> limsutils.select_proposal